module/VuFind/src/VuFind/ILS/Driver/MultiBackend.php

```diff
@@ -1042,11 +1042,11 @@ public function getAccountBlocks($patron)
      *
      * @return array An array with key-value pairs.
      */
-    public function getConfig($function, $params = null)
+    public function getConfig($function, $params = [])
```

Comment:
Just curious if @EreMaijala has any concerns about the changes to this file.

Reply:
I think the change is fine, but should be done for all ILS drivers.

Reply:
Very good point, @EreMaijala. @xmorave2, do you think it would make sense to create a separate PR to adjust getConfig for all drivers, rather than making this one bigger?

Reply:
Done in #2429

Reply:
Thanks, that looks good to me. I have merged it, so you can merge dev-9.0 into this branch whenever you're ready.